tilequeue/command.py

```diff
@@ -577,7 +577,8 @@ def tilequeue_process(cfg, peripherals):
     assert n_simultaneous_query_sets > 0
     # reduce queue size when we're rendering metatiles to try and avoid the
     # geometry waiting to be processed from taking up all the RAM!
-    default_queue_buffer_size = max(1, 128 >> (2 * (cfg.metatile_size or 0)))
+    default_queue_buffer_size = max(
+        1, 128 >> ((2 * (cfg.metatile_size or 0))) + 1)
```

Comment:
I think the original idea was that the data size is probably proportional to the query area, which should be proportional to the `metatile_size` squared, and therefore the queue length should be inversely proportional.

However, that seems to have been translated effectively into `128 / (2 ** (metatile_size * 2))`, which gives answers which are close enough. Rather than adding a `+1`, it could be written as `64 >> (metatile_size * 2)`, but might it be easier to read as follows?

```python
size_sqr = (cfg.metatile_size or 1)**2
default_queue_buffer_size = max(1, 16 / size_sqr)
```